graylog2-server/src/main/java/org/graylog2/rest/resources/HelloWorldResource.java

```diff
@@ -72,8 +74,9 @@ public HelloWorldResponse helloWorld() {
     @Produces({MediaType.TEXT_HTML, MediaType.APPLICATION_XHTML_XML})
     public Response redirectToWebConsole() {
         if (configuration.isRestAndWebOnSamePort()) {
+            final URI target = URI.create(configuration.getWebPrefix());
```

Comment:
I don't think this would work if the web interface and the Graylog REST API are listening on different network interfaces (e. g. different IP addresses and ports).